package.json

```
@@ -14,9 +14,9 @@
   },
   "packageManager": "yarn@4.0.2",
   "scripts": {
-    "dev": "echo 'coming soon'",
+    "dev": "vite build --watch && yarn storybook",
```

```suggestion
    "dev": "vite build --watch & yarn storybook",
```

Only one `&` to run in parallel instead of in sequence

---

I forgot to save the file :facepalm: